modules/microshift-gathering-sos-report.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+
+// * microshift_support/microshift-sos-report
+
+:_content-type: PROCEDURE
+[id="gathering-data-microshift-sos-report_{context}"]
+= Gathering data from a {product-title} sos report
+
+.Prerequisites
+
+* You must have the `sos` package installed.
+
+.Procedure
+
+. Log into the failing host with your credintials.
+
+. To view the available profiles, run the following command:
++
+[source,terminal]
+----
+$ sos report --list-profiles
+----
+//not sure if the above step is needed. Need verification.
+. Perform the debug report creation procedure by running the following command:
++
+[source,terminal]
+----
+$ sos report --batch --clean --all-logs --profile microshift
```

Comment:
`sos` needs to run as root, so we either need to say `sudo ...` here or we need to tell them to login as root up on line 15.